client/components/table/README.md

```diff
@@ -1,48 +1,88 @@
-Table
-=====
+`Table` Components
+==================
 
-This is an accessible, sortable, and scrollable table for displaying tabular data (like revenue and other analytics data). It accepts `headers` for column headers, and `rows` for the table content. `rowHeader` can be used to define the index of the row header (or false if no header). Sortability is enabled by default (and can be turned off), however only string and number columns are sortable.
+This is an accessible, sortable, and scrollable table for displaying tabular data (like revenue and other analytics data). It accepts `headers` for column headers, and `rows` for the table content. `rowHeader` can be used to define the index of the row header (or false if no header).
+
+`TableCard` serves as Card wrapper & contains a card header, `<Table />`, `<TableSummary />`, and `<Pagination />`. This includes filtering and comparison functionality for report pages.
+
+`Table` itself can be used outside of the Card + filtering context for displaying any tabular data.
 
 ## How to use:
 
 ```jsx
-import Table from 'components/table';
+import { TableCard } from 'components/table';
 
 render: function() {
   return (
-    <Table
+    <TableCard
+      title="Revenue Last Week"
       rows={ rows }
       headers={ headers }
-      caption={ __( 'Revenue Last Week' ) }
+      onQueryChange={ this.onQueryChange }
+      query={ query }
+      summary={ summary }
     />
   );
 }
 ```
 
 ## Props
 
+### `TableCard` props
+
+* `headers`: An array of column headers
+* `onQueryChange`: A function which returns a callback function to update the query string for a given `param`
+* `onClickDownload`: A callback function which handles then "download" button press.
```

Comment:
Looks like this could be set to null/falsey to disable/not render the download button correct?

Reply:
Yep- I had even made that update locally, but didn't push the updated branch 😶